src/sentry/issues/escalating_group_forecast.py

```diff
@@ -40,11 +40,11 @@ def save(self) -> None:
         nodestore.set(
             self.build_storage_identifier(self.project_id, self.group_id),
             self.to_dict(),
-            ttl=timedelta(TWO_WEEKS_IN_DAYS_TTL),
+            ttl=timedelta(ONE_WEEKS_IN_DAYS_TTL),
```

Comment:
It might be wise to keep this at 2 weeks. Then if the job is delayed at least we have data in place.

I'd also recommend renaming `TWO_WEEKS_IN_DAYS_TTL` to `GROUP_FORECAST_TTL`, that way you don't need to rename it whenever the value change.

Reply:
good point, thanks for bringing this up